src/librustdoc/html/render/mod.rs

```diff
@@ -1370,13 +1385,40 @@ impl Setting {
                 if *default_value { " checked" } else { "" },
                 description,
             ),
+            Setting::Select {
+                ref js_data_name,
+                ref description,
+                ref default_value,
+                ref options,
+            } => format!(
+                "<div class='setting-line'>\
```

Comment:
I think you might want to use `"` rather than `'`, I recently just send a patch on that.

Reply:
Why? As far as I know, they're equivalent and don't require escaping here.

Reply:
It's about unification. Nothing more. Please do so then. ;)

Reply:
Oh alright! Should I also change all the other simple quotes that were already used in the code?

Reply:
iirc, @pickfire has a PR ready for them, so just do it for the ones you added please.